File: core/src/main/java/com/sys1yagi/swipe/core/entity/swipe/Font.java
package com.sys1yagi.swipe.core.entity.swipe;

import com.google.gson.annotations.SerializedName;

public class Font {

    @SerializedName("size")
    int size;

    @SerializedName("name")
    String name;

    public int getSize() {
        return size;
    }

    public String getName() {
        return name;
    }
}

File: core/src/main/java/com/sys1yagi/swipe/core/entity/swipe/Font.kt
package com.sys1yagi.swipe.core.entity.swipe

import com.google.gson.annotations.SerializedName

class Font() {

    constructor(name: String?, size: Int) : this() {
        this.name = name
        this.size = size
    }

    @SerializedName("size")
    var size: Int = 0

    @SerializedName("name")
    var name: String? = null
}
